src/Components/Reusable/InAppBrowser/BrowserBottomSheet.spec.tsx

```diff
@@ -0,0 +1,196 @@
+import React from "react"
+import { COLORS } from "~Constants"
+
+jest.mock("@react-navigation/stack", () => ({
+    createStackNavigator: jest.fn(() => ({
+        Navigator: ({ children }: { children: React.ReactNode }) => children,
+        Screen: ({ children }: { children: React.ReactNode }) => children,
+        Group: ({ children }: { children: React.ReactNode }) => children,
+    })),
+}))
+
+jest.mock("@react-navigation/native-stack", () => ({
+    createNativeStackNavigator: jest.fn(() => ({
+        Navigator: ({ children }: { children: React.ReactNode }) => children,
+        Screen: ({ children }: { children: React.ReactNode }) => children,
+    })),
+}))
+
+jest.mock("@react-navigation/material-top-tabs", () => ({
+    createMaterialTopTabNavigator: jest.fn(() => ({
+        Navigator: ({ children }: { children: React.ReactNode }) => children,
+        Screen: ({ children }: { children: React.ReactNode }) => children,
+    })),
+}))
+
+jest.mock("@react-navigation/bottom-tabs", () => ({
+    createBottomTabNavigator: jest.fn(() => ({
+        Navigator: ({ children }: { children: React.ReactNode }) => children,
+        Screen: ({ children }: { children: React.ReactNode }) => children,
+    })),
+    useBottomTabBarHeight: jest.fn(() => 10),
+}))
+
+jest.mock("react-native-view-shot", () => ({
+    captureRef: jest.fn(),
+    releaseCapture: jest.fn(),
+}))
+
+jest.mock("~Components/Providers/InAppBrowserProvider", () => ({
+    useInAppBrowser: () => ({
+        isDapp: true,
+        navigationState: { url: "https://example.com", canGoBack: true },
+        webviewRef: { current: { reload: jest.fn(), goBack: jest.fn() } },
+        dappMetadata: { name: "Test", description: "Test", url: "https://example.com" },
+    }),
+}))
+
+jest.mock("~Components/Providers/FeatureFlagsProvider", () => ({
+    useFeatureFlags: () => ({
+        betterWorldFeature: { appsScreen: { enabled: true } },
+    }),
+}))
+
+jest.mock("~Hooks/useDappBookmarking", () => ({
+    useDappBookmarking: () => ({
+        isBookMarked: false,
+        toggleBookmark: jest.fn(),
+    }),
+}))
+
+jest.mock("@react-navigation/native", () => ({
+    useNavigation: () => ({ replace: jest.fn() }),
+}))
+
+jest.mock("~Storage/Redux", () => ({
+    useAppDispatch: () => jest.fn(),
+    useAppSelector: () => "test-tab-id",
+    closeTab: jest.fn(),
+    selectCurrentTabId: jest.fn(),
+}))
+
```

Comment:
Do you really need to mock all of these just to test if the color is ok?
The usual approach, at least for me, would be to mock as less as possible, unless you need to spy on that function call or mock the result to show a different UI/act on a different method

Reply:
Addressed.